gen/dcos-config.yaml

```diff
@@ -3028,6 +3063,8 @@ package:
       CALICO_IPV4POOL_NAT_OUTGOING=true
       FELIX_IGNORELOOSERPF=true
       FELIX_LOGFILEPATH=none
+      FELIX_PROMETHEUSMETRICSENABLED=true
+      FELIX_PROMETHEUSMETRICSPORT=62091
```

Comment:
a hard-coded port is used here because
1. 62091 is not in the Mesos port arrange 
2. we don't provide a preflight check for a dynamic port/configuration in https://github.com/dcos/dcos/pull/6709/files#diff-bdb9650f6145f942a2f0f88fe0329ddaR519